src/pages/Login/Input.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Input = ({ title, name, placeholder, detail, type, value, onChange }) => {
+  return (
+    <InputWrap>
+      <Name value={value}>{title}</Name>
+      <InfoInput
+        placeholder={placeholder}
+        type={type}
+        name={name}
+        value={value}
+        onChange={onChange}
+      />
+      {value.length > 0 ? <Validation>{detail}</Validation> : null}
```

Comment:
```suggestion
      {value.length > 0 && <Validation>{detail}</Validation>}
```
3항연산자는 뒤에 값이 있을 때 사용하는 것이 더 적합합니다!
이런 상황에서는 && 연산자를 사용해주세요~